workers/unity/Packages/io.improbable.gdk.mobile/Editor/iOSLaunchUtils.cs

```diff
@@ -90,8 +104,7 @@ public static void Build(string developmentTeamId)
 
                 if (string.IsNullOrEmpty(developmentTeamId))
                 {
-                    Debug.LogError("Development Team Id was not specified. Unable to build the XCode project.");
-                    return;
+                    Debug.LogWarning("Development Team Id was not specified. The build will fail, if the XCode project needs to build for device.");
```

Comment:
Don't we know whether we are targeting device/simulator? Can we early exit instead?

Reply:
it is surprisingly difficult to check whether an xcode project is for simulator or device (cause usually you should be able to build it for both). for unity, we can do a hack and check for which architecture the`libiPhone-lib` file has been built for. will add that